modules/RTC/RTCUtils.js

```diff
@@ -243,9 +243,7 @@ function getConstraints(um, options = {}) {
             }
 
             // Old style.
```

Comment:
Does the "old style" comment still apply? Does new style also need a mandatory video?

Reply:
I was not sure about the comment, what does it mean? 
I can remove it.

Reply:
i do think this comment still applies, but it doesn't strike me as a very useful comment even if that is the case.

Reply:
I looked at the isNewStyleConstraintsSupported flow some more. In short, it may not need supporting for this change. This getConstriants function should be part of old gum flow. isNewStyleConstraintsSupported includes firefox, which shouldn't be using old gum (btw, camera switching doesn't work anymore on firefox...). isNewStyleConstraintsSupported includes edge which hasn't been able to device switch according to the ortc readme, and it includes temasys, which is pretty much going to be no longer supported. That leaves react native under isNewStyleConstraintsSupported, but I think it react native uses the option facingMode instead of cameraDeviceId.

Reply:
Hum, switching camera for FF works for me, but it is sooo slow and you need to wait.

Reply:
Yep performance overall, performance analyses show paint and graphics taking a lot of time.

Reply:
For my reference: Off Main Thread Paint (OMTP) was introduced to Mac FireFox with 59. I just checked ESR (52) and it's nowhere near as sluggish as 59.